codersdk/deployment.go

```diff
@@ -2340,6 +2357,18 @@ func (c *DeploymentValues) Options() serpent.OptionSet {
 			YAML:        "maxTokenLifetime",
 			Annotations: serpent.Annotations{}.Mark(annotationFormatDuration, "true"),
 		},
+		{
+			Name: "Max Token Lifetime Expression",
+			Description: "A CEL expression that determines the maximum token lifetime based on user attributes. " +
+				"The expression has access to 'subject' (rbac.Subject), 'requestedLifetime' (string), 'globalMaxDuration' (string), " +
+				"'defaultDuration' (string), and 'tokenScope' (string). " +
+				"Must return a duration string (e.g., duration(\"168h\")). " +
+				"Example: 'subject.roles.exists(r, r.name == \"owner\") ? duration(globalMaxDuration) : duration(defaultDuration)'.",
```

Comment:
Suggest adding a link to the CEL documentation here so it shows in the help output and the docs.